pxtcompiler/emitter/snippets.ts

```diff
@@ -118,6 +120,14 @@ namespace ts.pxtc.service {
         let addNamespace = false;
         let namespaceToUse = "";
         let functionCount = 0;
+        let parentSnippet: SnippetNode;
+        let overrideParamLabel: string;
+        let overrideSnippet: SnippetNode;
+        if (parameterOverride) {
+            overrideParamLabel = Object.keys(parameterOverride)?.[0];
+            overrideSnippet = parameterOverride[overrideParamLabel];
```

Comment:
Should we check that `overrideParamLabel` is nonnull?

Reply:
Sure!